src/cephadm/cephadm

```diff
@@ -2004,6 +2043,20 @@ def command_bootstrap():
 
 ##################################
 
+def extract_uid_gid_monitoring(daemon_type):
+    # type: (str) -> Tuple[int, int]
+
+    if daemon_type == 'prometheus':
+        uid, gid = extract_uid_gid(file_path='/etc/prometheus')
+    elif daemon_type == 'node-exporter':
+        uid, gid = 65534, 65534
+    elif daemon_type == 'grafana':
+        uid, gid = extract_uid_gid(file_path='/var/lib/grafana')
+    else:
+        raise Error("{} not implemented yet".format(daemon_type))
+    return uid, gid
+
```

Comment:
Maybe we can make the other way around? What about calling `extract_uid_gid_monitoring` from `extract_uid_gid`?

Reply:
and/or lift this into the `Monitoring` class .. ?